scripts/setonix-cpu.profile

```diff
@@ -7,12 +7,13 @@ module load cray-mpich
 module load cce/15.0.1
 module load cray-hdf5
 module load cray-python/3.9.13.1
+module load rocm/5.4.3
 
 # GPU-aware MPI
 export MPICH_GPU_SUPPORT_ENABLED=0
 
 # compiler environment hints
 export CC=$(which cc)
-export CXX=$(which CC)
+export CXX="$(which CC) -fno-cray"
```

Comment:
@BenWibking will this fix #566 for CPUs? Will it work even if the compiler is not cray?

Reply:
It's probably a better idea to detect the cray compiler based on the value of `CMAKE_CXX_COMPILER_ID`, e.g.
https://github.com/quokka-astro/quokka/blob/a2acbb741a282be88b48ad52279196c3d421aaeb/src/CMakeLists.txt#L79
except the appropriate compiler ID is "CrayClang" (https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html).

Reply:
oops, put it in the wrong place